packages/fx-core/resource/package.nls.json

```diff
@@ -312,8 +312,8 @@
   "core.createProjectQuestion.capability.customCopilotBasicOption.detail": "Build a basic AI chatbot in Teams",
   "core.createProjectQuestion.capability.customCopilotRagOption.label": "Chat With Your Data",
   "core.createProjectQuestion.capability.customCopilotRagOption.detail": "Expand AI bot's knowledge with your content to get accurate answers to your questions",
-  "core.createProjectQuestion.capability.customCopilotAssistantOption.label": "AI Assistant",
-  "core.createProjectQuestion.capability.customCopilotAssistantOption.detail": "Build intelligent chatbot in Teams to complete everyday tasks efficiently",
+  "core.createProjectQuestion.capability.customCopilotAssistantOption.label": "AI Agent",
+  "core.createProjectQuestion.capability.customCopilotAssistantOption.detail": "Build an AI agent in Teams that can reason over LLMs to take a sequence of actions",
```

Comment:
This is complex and will add cognitive load to users.
Proposed: Build an AI agent in Teams that can make decisions and perform actions based on LLM reasoning.

Reply:
Agree with you. Thanks, updated.